gpiozero/boards.py

```diff
@@ -102,6 +69,61 @@ def toggle(self):
         for led in self.leds:
             led.toggle()
 
+
+
+class LEDBoard(LEDCollection):
+    """
+    Extends :class:`CompositeDevice` and represents a generic LED board or
+    collection of LEDs.
+
+    The following example turns on all the LEDs on a board containing 5 LEDs
+    attached to GPIO pins 2 through 6::
+
+        from gpiozero import LEDBoard
+
+        leds = LEDBoard(2, 3, 4, 5, 6)
+        leds.on()
+
+    :param int \*pins:
+        Specify the GPIO pins that the LEDs of the board are attached to. You
+        can designate as many pins as necessary.
+
+    :param bool pwm:
+        If ``True``, construct :class:`PWMLED` instances for each pin. If
+        ``False`` (the default), construct regular :class:`LED` instances. This
+        parameter can only be specified as a keyword parameter.
+    """
+
+    def close(self):
+        self._stop_blink()
+        super(LEDBoard, self).close()
+
+    @property
+    def value(self):
+        """
+        A tuple containing a value for each LED on the board. This property can
+        also be set to update the state of all LEDs on the board.
+        """
+        return tuple(led.value for led in self._leds)
+
+    @value.setter
+    def value(self, value):
+        self._stop_blink()
+        for l, v in zip(self.leds, value):
+            l.value = v
+
+    def on(self):
+        self._stop_blink()
+        super(LEDBoard, self).on()
+
+    def off(self):
+        self._stop_blink()
+        super(LEDBoard, self).off()
+
+    def toggle(self):
+        self._stop_blink()
+        super(LEDBoard, self).toggle()
+
     def blink(
```

Comment:
I've been trying to work out what mathematical operation LEDBarGraph.toggle() would be equivalent to, and I *think* it's:
```python
import math
self.value -= math.copysign(1,self.value)
```
The above technically converts 0 to -1 (rather than 1), but that's not a problem as for the LEDBarGraph 1 and -1 are indistinguishable.